.github/workflows/publish-sandbox-ui-for-ui-e2e-tests.yml

```
@@ -0,0 +1,100 @@
+                      \"auths\": {
+                              \"quay.io\": {
+                                      \"auth\": \"${{ secrets.TEST_QUAY_TOKEN }}\"
+                              }
+                      }
+                  }"> ${XDG_RUNTIME_DIR}/containers/auth.json
+          podman login --get-login quay.io
+
+      # Publish Developer Sandbox UI for PR events
+      - name: Publish current Developer Sandbox UI (PR event)
+        if: ${{ github.event_name == 'pull_request_target' }}
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)